File: csharp/VS2005/netframework/AboutForm.cs
using System;
using System.Drawing;
using System.Collections;
using System.ComponentModel;
using System.Windows.Forms;
using System.Reflection;
using FlexCel.Core;

namespace MainDemo
{
    /// <summary>
    /// About...
    /// </summary>
    public partial class AboutForm: System.Windows.Forms.Form
    {

        public AboutForm()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, System.EventArgs e)
        {
            Close();
        }

        private void linkLabel1_LinkClicked(object sender, System.Windows.Forms.LinkLabelLinkClickedEventArgs e)
        {
            System.Diagnostics.Process.Start(linkLabel1.Text);
        }

        private void linkLabel2_LinkClicked(object sender, System.Windows.Forms.LinkLabelLinkClickedEventArgs e)
        {
            System.Diagnostics.Process.Start(linkLabel2.Text);
        }

        private void AboutForm_Load(object sender, System.EventArgs e)
        {
            Assembly asm = Assembly.GetAssembly(typeof(ExcelFile));
            lblVersion.Text = "Using FlexCel Version: " + asm.GetName().Version.ToString();

        }
    }
}

File: vb/VS2019/netframework/AboutForm.vb
Imports System.Collections
Imports System.ComponentModel
Imports System.Reflection
Imports FlexCel.Core

Namespace MainDemo
	''' <summary>
	''' About...
	''' </summary>
	Partial Public Class AboutForm
		Inherits System.Windows.Forms.Form

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub button1_Click(ByVal sender As Object, ByVal e As System.EventArgs) Handles button1.Click
			Close()
		End Sub

		Private Sub linkLabel1_LinkClicked(ByVal sender As Object, ByVal e As System.Windows.Forms.LinkLabelLinkClickedEventArgs) Handles linkLabel1.LinkClicked
			System.Diagnostics.Process.Start(linkLabel1.Text)
		End Sub

		Private Sub linkLabel2_LinkClicked(ByVal sender As Object, ByVal e As System.Windows.Forms.LinkLabelLinkClickedEventArgs) Handles linkLabel2.LinkClicked
			System.Diagnostics.Process.Start(linkLabel2.Text)
		End Sub

		Private Sub AboutForm_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.Load
			Dim asm As System.Reflection.Assembly = System.Reflection.Assembly.GetAssembly(GetType(ExcelFile))
			lblVersion.Text = "Using FlexCel Version: " & asm.GetName().Version.ToString()

		End Sub
	End Class
End Namespace
